src/index.js

```diff
@@ -1,6 +1,9 @@
 const fibonacci = (n) => {
-
+    let arreglo = [];
+    for(let i = 0; i < n; i++){
+      (i === 0) ? arreglo.push(1) : arreglo.push(arreglo[i-2] + arreglo[i-1] || 1)
```

Comment:
Prioritize that your code is uderstandable before it is short.

Reply:
Thank you. I just rewrited the code for better legibility.